web/src/pages/apd/activities/cost-allocation/CostAllocateFFPQuarterly.js

```diff
@@ -148,17 +144,20 @@ const CostAllocateFFPQuarterly = ({
                   <Controller
                     control={control}
                     name={`[${q}].inHouse.percent`}
-                    render={({ field: { onBlur, ...props } }) => (
+                    value={quarterlyFFP[year][q].inHouse.percent}
```

Comment:
you don't really need to do this, it's probably better to change it back to how it was before